src/services/embargo.ts

```diff
@@ -158,83 +168,88 @@ export function calculateEmbargoStage(dbFile: File): EmbargoStage {
  *  it does not rely on the state of the file in the DB except for clincial exemption.
  */
 export function calculateEmbargoStartDate(inputs: {
-  dbFile: File;
-  songAnalysis: SongAnalysis;
-  matchedSamplePairs: MatchedSamplePair[];
-  clinicalDonor?: ClinicalDonor;
+	dbFile: File;
+	songAnalysis: SongAnalysis;
+	matchedSamplePairs: MatchedSamplePair[];
+	clinicalDonor?: ClinicalDonor;
 }): Date | undefined {
-  const { dbFile, songAnalysis, matchedSamplePairs, clinicalDonor } = inputs;
+	const { dbFile, songAnalysis, matchedSamplePairs, clinicalDonor } = inputs;
 
-  // Check for clinical exemption, if no exemption then we need valid clinical data
-  const clinicalExemption: boolean = dbFile.clinicalExemption !== undefined;
-  const validClinicalData: boolean = clinicalDonor?.schemaMetadata?.isValid || false;
-  if (!clinicalExemption) {
-    if (!clinicalDonor) {
-      logger.info(
-        `calculateEmbargoStartDate()`,
-        dbFile.fileId,
-        `file ${dbFile.fileId} has no clinical donor data available and has no clinical exemption.`,
-      );
-      return undefined;
-    }
-    if (!validClinicalData) {
-      logger.info(
-        `calculateEmbargoStartDate()`,
-        dbFile.fileId,
-        `file ${dbFile.fileId} has a donor with invalid clinical data and has no clinical exemption.`,
-      );
-      return undefined;
-    }
-  }
+	// Check for clinical exemption, if no exemption then we need valid clinical data
+	const clinicalExemption: boolean = dbFile.clinicalExemption !== undefined;
+	const clinicalCoreComplete: boolean = clinicalDonor?.completionStats?.coreCompletionPercentage === 1 || false;
```

Comment:
This line is the only functional change.

Reply:
can just be:
```js
	const clinicalCoreComplete: boolean = clinicalDonor?.completionStats?.coreCompletionPercentage === 1;``` 
?

Reply:
Yeup, guess I carried over the formatting from the previous version without thought. Will clean it up like this.